src/boot/efi/random-seed.c

```diff
@@ -168,7 +164,6 @@ EFI_STATUS acquire_system_token(VOID **ret, UINTN *ret_size) {
 }
 
 static VOID validate_sha256(void) {
-
 #ifndef __OPTIMIZE__
```

Comment:
Please drop this `#ifndef`. We generally don't want to change behaviour based on complication flags. The check should be very quick, so it is not a problem to enable it uncondtitionally.

Reply:
That defeats the purpose of making the build actually optizimed and will reintroduce the message we want to get rid of in the first place. This just shouldn't be in production builds.

Reply:
> i mean, the msg is not the problem, we shouldn't validate SHA256 at all if we are running a production build

Please don't run unit tests in production builds.